3_visualize/log/daily_timeseries_summary.csv

```diff
@@ -0,0 +1,112 @@
+site_no,n_obs,mean,sd
+01411400,2,141,0
```

Comment:
Did you only save sites with 2 or more data points? sd would not exist for 1  point, so I'm wondering if all of those sites got omitted from the log file as a result

Reply:
This summary file (`3_visualize/log/daily_timeseries_summary.csv`) is for the daily NWIS data. All daily sites that get returned have at minimum 2 days reported, so I have not done any additional filtering here. 

The discrete data would include examples where there is only 1 data point for a given site. However, I haven't saved/committed a similar summary file for the discrete data since it is static (apart from periodic updates) and less subject to changes.

Reply:
Does discrete data here mean qw from NWIS, or are there other sources? Is there a non-timeseries summary for these discrete data or are they otherwise tracked? For example, we could add total number of discrete samples and sites to `3_visualize/log/sitelist_summary.csv`.

Reply:
Yes, discrete data would mean "qw" data for USGS-sites, but for the sake of our project discrete data means data from the harmonized WQP dataset (which includes the NWIS water quality data). Right now, the discrete data are tracked with an indicator file w/ a data hash (`3_visualize/log/wqp_data_ind.csv`). I didn't add more summary information from the harmonized WQP data because I don't expect that dataset to be updated very often. 

The number of sites in the filtered WQP dataset is included in `3_visualize/log/sitelist_summary.csv`. Right now we only return the total observation-days across discrete and continuous measurements, but we could add a column that indicates the number of discrete-only observation-days. I'm curious if you were able to make sense of the `sitelist_summary`. Some sites intersect the discrete data, daily data, and instantaneous data, so my approach was to give 'total' numbers as well as numbers that reflect unique site id's/latlon. The `n_allsites_nwis` columns would be the columns to check to see whether any new "dv" or "uv" sites were added in a given pipeline build.

Reply:
Okay, that makes sense for not tracking discrete harmonized WQP data. I think the data hash is sufficient. 

Yes, I think I followed the logic you used for columns to report in the `sitelist_summary`. It would be helpful to track discrete only observations. This could be important once the updated to 2021 WQP data are pulled into this workflow.